.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -19,9 +19,22 @@ Steps to reproduce the behavior.
 **Expected behavior**
 A clear and concise description of what you expected to happen.
 
+**Logs**
+Check if any warnings or errors were logged by VictoriaMetrics components
+or components in communication with VictoriaMetrics (e.g. Prometheus, Grafana).
+
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
+For VictoriaMetrics health-state issues please provide full-length screenshots 
```

Comment:
Should we introduce another template for issues which require screenshot from dashboards?
only few of them https://github.com/VictoriaMetrics/VictoriaMetrics/issues?q=is%3Aopen+is%3Aissue+label%3Abug are required of data from dashboards

Reply:
I was thinking to split templates into `queries and visualisation` and `system health and stability`. But decided it will be too complicated. What do you think?

Reply:
It is better from simplicity PoV to use a single issue template